definitions/actions.lua

```diff
@@ -278,6 +278,7 @@ return {
       PrevTransientInItem = {"SaveItemSelection", "SelectItemsUnderEditCursor", "MoveEditCursorToPrevTransientInSelectedItems", "RestoreItemSelection"},
       ProjectEnd = custom.move.projectEnd,
       ProjectStart = custom.move.projectStart,
+      ProjectSettings = 40021,
```

Comment:
Hey thanks for doing this! I would rename ProjectSettings to 'ShowProjectSettings' though, nouns are used for selecting tracks / parts of the timeline. Take a look at the top of actions.lua to see some notes about naming conventions.

Once you change that it should be all good

Reply:
Yep! I'll fix that right up. Also, I wonder if ShowProjectSettings would be more useful in the <SPC> project folder? Or do you prefer it where it is now?

Reply:
Yeah I'd put it in the project folder, probably under '<SPC>p,'!

Reply:
'p' Is taken there by PreviousTab :/

Reply:
'P'??

Reply:
Oh wait... Hmmm. There seem to be two I stances of PrevTab. One is 'N' the other is 'P'